INSTALLATION.md

```diff
@@ -35,45 +35,59 @@ cp .env.template .env
 
 ```shell
 POSTGRES_DB # Name of the database
-POSTGRES_USER # Your local postgres username
-POSTGRES_PASSWORD # Your local postgres password
+POSTGRES_USER # A username to access the database
+POSTGRES_PASSWORD # A password to access the database
 SECRET_KEY # A secret key to hash your application
 BINANCE_API_KEY # Your personal Binance API key (check binance documentation)
 BINANCE_API_SECRET # Your personal Binance API secret (check binance documentation)
 BINANCE_API_KEY_TEST # Your personal Binance API key (binance testnet)
 BINANCE_API_SECRET_TEST # Your personal Binance API secret (binance testnet)
 ```
 
+
+In order to run the bot locally, you'll need to have **`docker`** installed. You can follow the instructions [here](https://docs.docker.com/get-docker/) to install it.
+If you are on a linux based machine, verify that you can run docker commands without `sudo` privileges. If not, make sure you do the steps detailed in the [post-installation](https://docs.docker.com/engine/install/linux-postinstall/) page.
+
 ### Database
 
 This project uses PostgreSQL as a database engine in order to keep consistency between local development and production.
-Therefore, it is assumed that PostgreSQL is installed on your machine. If not, please visit this
-[page](https://www.postgresql.org/download/).
 
-We must first create the database by entering into `psql` and executing a command there:
+We can start the services in detached mode via `docker-compose`, as follows:
 ```shell
-psql -d postgres -U <pg_username> # Replace <pg_username> with your own postgres username
-CREATE DATABASE <database_name>; # Enter the same name as the env variable $POSTGRES_DB
-exit
+docker-compose up -d --build
 ```
-In order to initialize the database, run the following commands:
+This allows us to access the containers to perform commands to create the database and make the migrations.
+
+In order to create the database, we can enter the database container as follows:
 ```shell
-python database/manage.py migrate
-python database/initial_setup.py
+docker-compose exec db bash
 ```
-This will load the schema onto the database and load some required initial data 
-from binance. You can then create a superuser with the following command and follow the
-instructions:
+Once inside the container, we can enter the psql console, check if the database is already created, and if not, create it.
 
+to create the database
 ```shell
-python database/manage.py createsuperuser
+psql -U postgres
+\l+ # Will list all the databases. Check if there is a database present with the name you specified in the env variables. 
+CREATE DATABASE <database_name>;  # Create the db. Skip this if the db is already there.
 ```
```

Comment:
This shouldn't be needed, since the database and the user are automatically created from the environment variables passed to the docker-compose file.

The correct command to run inside the db container is `psql -U $POSTGRES_USER $POSTGRES_DB`, getting the correct values directly from the container env variables, otherwise if the .env has different values you get errors like `psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  role "postgres" does not exist`.

In my opinion you can safely skip and remove this step, and just tell the user to make sure they filled all the `.env` file entries.

Reply:
Yeah, I also noticed that the database was created automatically, even if it was only on the first time I spawned the service. Still I feel that it is probably nice to reference how one would access the container and the psql console if needed.